projects/helper/portedTokens.js

```diff
@@ -2061,6 +2075,7 @@ const chainTransforms = {
   reichain: transformReichainAddress,
   nova: transformNovachainAddress,
   canto: transformCantoAddress,
+  telos: transformTelosAddress(),
```

Comment:
think this should be `telos: transformTelosAddress,`

Reply:
sorry, my mistake.

fixed